packages/docs/src/examples/va-image/DefaultSlot.vue

```diff
@@ -1,30 +1,27 @@
 <template>
   <div class="row">
     <div class="flex md6 lg4">
-      <va-image
-        style="min-height: 300px;"
-        src="https://picsum.photos/1500"
-      >
-        <va-badge text="default slot" />
+      <va-image style="min-height: 300px;" src="https://picsum.photos/1500">
+        <va-badge :text="$t('api.VaImage.examples.defaultSlot')" />
```

Comment:
`default slot` you can leave the way it is. Hardly translatable.

Reply:
left `default slot` without translation